### DISCUSSION TITLE
在版本设置里的快捷方式区域添加更多功能

### DISCUSSION THREAD

**Discussion by Skying-zhishi** [Meloong-Git/PCL#938] [提问] [answered]
就比如光影文件夹，模组文件夹材质包文件夹之类的（而且只有能装的时候才显示）

**Comment by WorldHim** [comment]
这个需要判断有没有相应mod提供相应快捷方式的~~也太麻烦~~没什么必要吧，毕竟打开版本文件夹已经够用，~~就算龙猫会接受估计也是暂缓处理~~如果有需要建议通过自定义主页实现。
还有这种新功能反馈能不能发issue啊啊啊啊啊啊啊啊

**Reply by WForst-Breeze** [reply]
草（）可能大部分反馈交 Discussion 的都是不熟悉操作流程的朋友们，跟他们说一声就好了。说句不好听的就是，只要我们提醒到位了，交不交就是他们的事了（摊手

**Comment by WForst-Breeze** [comment, marked as answer]
已有相关 issue 提及此事，请见 #606
